README.md

```diff
@@ -12,7 +12,9 @@ Supports PHP 5.6 though PHP 7.6.
 # Usage
 
 ```php
-require('crc32.php');
+include "vendor/autoload.php";
```

Comment:
This may be dumb questions, but:

why include, not require?

also single quota strings, instead of double, to be consistent within the example?

Reply:
The include is just what I usually use. I swapped it out.

Reply:
Thanks